tests/dragonfly/proxy.py

```diff
@@ -0,0 +1,64 @@
+import asyncio
+import random
+
+
+class Proxy:
+    def __init__(self, host, port, remote_host, remote_port):
```

Comment:
that's genius as it provides introspection into what goes on between replica and master

Reply:
It is pretty neat indeed! And 90% of it is AI generated boilerplate, which is even more cool :laughing: